### ISSUE TITLE
Incorrect usage examples

### ISSUE THREAD

**Issue by adrianblynch** [alfredkam/bigData#5]
I had to use csv-to-json as follows:
```
var ctj = require("csv-to-json")

var json = ctj.parse({filename: "./bus-stops.csv", function (err, result) {
	console.log(err, result);
})
```

Not as the docs suggest:
```
var csv = require("./csv.js");
var json = csv.parse('path/to/file');
```

Have I misunderstood something?

**Comment by alfredkam** [comment]
That is indeed a mistake by me, I've forgot to update the documentation when ive made that change.  I'll fix it shortly.

**Comment by alfredkam** [comment]
fixed & published @0.1.0
